serverlessworkflow/modules/ROOT/pages/_common-content/getting-started-requirement.adoc

```diff
@@ -1,3 +1,3 @@
 * A workflow project is created.
 +
-For more information about creating a workflow project, see xref:getting-started/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service].
\ No newline at end of file
+For more information about creating a workflow project, see xref:getting-started/modules/create-your-first-workflow-service.adoc[Creating your first Serverless Workflow service].
```

Comment:
I would call this differently as otherwise we have `modules` twice in the path, which can cause confusion.